packages/frontend-shared/src/components/Input.cy.tsx

```diff
@@ -12,7 +12,7 @@ describe('<Input/>', { viewportWidth: 400, viewportHeight: 80 }, () => {
     cy.mount(() => <Input vModel={value.value}/>)
     cy.get('input').type(textToType, { delay: 0 })
 
-    cy.should(() => {
+    cy.wrap(null).should(() => {
```

Comment:
Why does `.wrap(null)` need to be added here?

Reply:
As part of this PR, I started enforcing our documented limits:

![image](https://user-images.githubusercontent.com/3003404/218808329-bd6a6c0f-0a85-4959-8223-c15317fe294c.png)

![image](https://user-images.githubusercontent.com/3003404/218808200-e7ae107b-5b0a-4e5c-a7d0-1164fd3abd1f.png)

Enforcing that isn't necessary - I can continue to allow `.should()` to be chained directly off of `cy` just fine. But it seemed like 'while already doing a breaking change around the command' was a good time to add the enforcement.

So I can do one of two things:
1. Make sure this is clear in the changelog and migration guide as a breaking change to match our documentation.
2. Remove this change from the PR, and update our docs to match the loose requirements.

Opinions?

Reply:
I pretty commonly use `cy.should()` just to retry functions that assert things, without a subject. Probably I'd do it less if I wasn't usually testing Cy internals, but still, I'm sure it happens in the wild too. 

Also, I think `cy.should()` having a default subject of `null` or `undefined` makes logical sense.

For these reasons, I think I'd prefer #2, since it avoids extra migration steps and it doesn't cause any logical inconsistency.

Reply:
Number 2 is good by me. :+1: 

The default subject will be `undefined` (unless someone can make strong case to the contrary, it would be a breaking change).

Reply:
Docs PR updated with this change - commands now no longer claim you must / can't chain them off `cy`, they call out what they require as a subject instead.

Also undid this test change and removed the code forcing it.